src/operator/timeout.ts

```diff
@@ -97,6 +93,6 @@ class TimeoutSubscriber<T> extends Subscriber<T> {
   }
 
   notifyTimeout(): void {
-    this.error(this.errorToSend);
+    this.error(new TimeoutError());
```

Comment:
@blesh because if you do end up needing it, the stack trace is awful if we don't do it at the constructor call site. See #2132